core-document-store/src/main/java/org/gluu/service/document/store/conf/DBDocumentStoreConfiguration.java

```diff
@@ -0,0 +1,90 @@
+package org.gluu.service.document.store.conf;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.io.Serializable;
+
+import org.gluu.persist.PersistenceEntryManager;
+
+/**
+ * @author shekhar L. on 27/04/2022
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DBDocumentStoreConfiguration implements Serializable {
+
+	private static final long serialVersionUID = 3380170170265842538L;
+
+	private String server; // http://localhost:8080
```

Comment:
We need to remove LDAP settings. DB document store should not depend on specific DB type